config/secrets.yml

```diff
@@ -0,0 +1,22 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
You don't want to push your `secrets.yml` file to a public repo! This is where you'll store sensitive information like API keys in the future.